feature-examples/wan/README.md

```diff
@@ -104,11 +104,73 @@ as well as printed to the console.
 
         Cluster-2 gfsh>query --query="select e.key, e.value from /example-region.entries e"
 
-11. Exit gfsh in each terminal and shutdown the cluster using the stop.gfsh script
+11. Feed data into regions
+        ../gradlew run
+
+12. Do wan-copy region with wild card
+        $ gfsh
+        ...
+        Cluster-2 gfsh>connect --locator=localhost[10332]
+
+        # Backward compatibility, previous wan-copy region command is still supported
```

Comment:
Since the wan-copy example is completely new we probably don't need to call this out as "backwards compatibility" and just highlight that the command can be used to specify a single region/sender id in addition to other behavior.

Reply:
Fixed.